medcat/config_meta_cat.py

```diff
@@ -48,27 +58,66 @@ class Config:
 class Model(MixingConfig, BaseModel):
     """The model part of the metaCAT config"""
     model_name: str = 'lstm'
-    """NOTE: When changing model, make sure to change the tokenizer as well"""
+    """
+    Model to be used for training or predicting.
+
+    Choose from:
+        - 'bert'
+        - 'lstm'
+
+    Note:
+        When changing the model, make sure to change the tokenizer accordingly.
+    """
     model_variant: str = 'bert-base-uncased'
+    """
+    Applicable only when using BERT:
+    
+    Specifies the model variant to be used.
+    """
     model_freeze_layers: bool = True
+    """
+    Applicable only when using BERT:
+    
+    Determines the training approach for BERT.
+
+    - If True: BERT layers are frozen and only the fully connected (FC) layer(s) on top are trained.
+    - If False: Parameter-efficient fine-tuning will be applied using Low-Rank Adaptation (LoRA).
+    """
     num_layers: int = 2
+    """Number of layers in the model (both LSTM and BERT)"""
     input_size: int = 300
+    """
+    Specifies the size of the embedding layer.
+
+    Applicable only for LSTM model and ignored for BERT as BERT's embedding size is predefined.
```

Comment:
This is not entirely true.

In `medcat.utils.meta_cat.models` (lines 93-94):
```
        if config.model['input_size'] != _bertconfig.hidden_size:
            logger.warning(f"\nInput size for {config.model.model_variant} model should be {_bertconfig.hidden_size}, provided input size is {config.model['input_size']} Input size changed to {_bertconfig.hidden_size}")
```
So it looks like there's some kind of expectation to what input size is set to.

Reply:
As it does not change the BERT's input size, we added the logger warning as a sign to notify that the given input size is not being used